app/models/scheme.rb

```diff
@@ -282,6 +282,7 @@ def status_at(date)
     return :deactivated if open_deactivation&.deactivation_date.present? && date >= open_deactivation.deactivation_date
     return :deactivating_soon if open_deactivation&.deactivation_date.present? && date < open_deactivation.deactivation_date
     return :reactivating_soon if last_deactivation_before(date)&.reactivation_date.present? && date < last_deactivation_before(date).reactivation_date
+    return :activating_soon if startdate.present? && date < startdate
```

Comment:
Can we have a test for this? Also should there be a validation that prevents adding deactivation periods before startdate? I assume we already do that for locations

Reply:
ah good point - I've added a test. The validation's already there - it works through available_from on both schemes and locations, which has been updated in this PR